docker/Dockerfile

```diff
@@ -257,7 +257,7 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 if [ "$TARGETPLATFORM" != "linux/arm64" ]; then \
     # TESTING: install FlashInfer from source to test 2.7.0 final RC
     FLASHINFER_ENABLE_AOT=1 TORCH_CUDA_ARCH_LIST='7.5 8.0 8.6 8.9 9.0+PTX' \
```

Comment:
do we need to update this to 10.0 as well?

Reply:
I wasn't sure if we would do that in this PR or the previous https://github.com/vllm-project/vllm/pull/18092